js/lesson_2/script_min.js

```diff
@@ -0,0 +1,17 @@
+let a = '0.1';
+let b = '0.2';
+
+console.log( (+a*10 + +b*10)/10);
+
+let c = '1';
+let d = 2;
+
+
+console.log( +c + d);
+console.log(Number(c) + d);
+
+
+const FILE =0.820;
+let memory =+prompt(`Введіть обє'єм памяті в ГБ `);
+// console.log ( memory % FILE );
```

Comment:
And you can use a variable for calcs, f.e.

```
let result = (memory - memory%FILE)/FILE)
console.log(result)
```

it will be more readable code